packet-solutions/tc_reply_kern_02.c

```diff
@@ -0,0 +1,65 @@
+/* SPDX-License-Identifier: GPL-2.0 */
+#include <linux/bpf.h>
+#include <linux/in.h>
+
+#include <linux/if_ether.h>
+#include <bpf/bpf_helpers.h>
+#include <bpf/bpf_endian.h>
+#include <linux/pkt_cls.h>
+
+// The parsing helper functions from the packet01 lesson have moved here
+#include "../common/parsing_helpers.h"
+
+SEC("fix_port_egress")
+int _fix_port_egress(struct __sk_buff *skb)
+{
+	struct iphdr *iphdr;
+	struct ipv6hdr *ipv6hdr;
+        struct udphdr *udphdr;
+        struct tcphdr *tcphdr;
+	int eth_type, ip_type;
+	void *data = (void *)(long)skb->data;
+	struct ethhdr *eth;
+	void *data_end = (void *)(long)skb->data_end;
+	struct hdr_cursor nh = { .pos = data };
```

Comment:
We follow the netdev "reverse christmas tree" variable ordering, see https://docs.kernel.org/process/maintainer-netdev.html#local-variable-ordering-reverse-xmas-tree-rcs

Please reorder these accordingly...